PWGPP/EvTrkSelection/AliAnalysisTrackingUncertaintiesAOT.cxx

```diff
@@ -138,7 +140,9 @@ AliAnalysisTrackingUncertaintiesAOT::AliAnalysisTrackingUncertaintiesAOT(const c
   fCutGeoNcrNclLength(130.),
   fCutGeoNcrNclGeom1Pt(1.5),
   fCutGeoNcrNclFractionNcr(0.9),
-  fCutGeoNcrNclFractionNcl(0.7)
+  fCutGeoNcrNclFractionNcl(0.7),
+  fWhichCuts(kDefault),
+  fTPCclstCut(0)
```

Comment:
Ciao Mattia,
shouldn't we use as default fTPCclstCut(1)?
In this case, when using the standard TPCITS 2011 cuts we use the selection:
     esdTrackCuts->SetMinNCrossedRowsTPC(70);
    esdTrackCuts->SetMinRatioCrossedRowsOverFindableClustersTPC(0.8);
(see below).

Ciao, Francesco